### PR TITLE
WIP chore(deps): lock version 3.0.3

### PR DESCRIPTION
**Pull request by Wxh16144**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [x] Workflow
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |     --      |
| 🇨🇳 Chinese |     --      |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

---

### <samp>🤖 Generated by Copilot at a04729d</samp>

Fixed a dependency version in `package.json` to prevent test failures caused by a breaking change in `scroll-into-view-if-needed`.

### <samp>🤖 Generated by Copilot at a04729d</samp>

* Fix dependency version of `scroll-into-view-if-needed` to avoid breaking changes ([link](https://github.com/ant-design/ant-design/pull/41715/files?diff=unified&w=0#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519L156-R156))

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -153,7 +153,7 @@
     "rc-trigger": "^5.3.4",
     "rc-upload": "~4.3.0",
     "rc-util": "^5.27.0",
-    "scroll-into-view-if-needed": "^3.0.3",
+    "scroll-into-view-if-needed": "3.0.3",
```
Comment (yoyo837) [inline package.json:156]: 啥问题

Comment (Wxh16144) [reply package.json:156]: 这个包导致 CI 跑 dist 打包的时候有问题，我还在看，这个是最粗暴的解决方法。

我参考了这个包的 更新日志，还没找到其他解决方案 :(

Comment (li-jia-nan) [reply package.json:156]: 问题是 3.0.8 出现的，锁到 3.0.7 试试会不会有问题？

**Comment by li-jia-nan** [comment]
应该不用锁了，我看到那个库回滚了

File: package.json (hunk 1 above)
Comment (li-jia-nan) [reply package.json:156]: 应该不用锁了，我看到那个库回滚了

**Pull request closed without merging**